characters.py

```diff
@@ -128,6 +128,28 @@ def heal(self, other):
         """
         min(other.current_hp + self.healing_power, other.max_hp)
 
+
+class Voodoo(Character):
+    def __init__(self, poison_nova, name):
+        Character.__init__(self, 1000, name)
+        self.poison_nova = poison_nova
+
+    def act(self, other):
+        self.poison(other)
+        self.check_if_dead(other)
+
+    def poison(self, other):
+        if not self.check_if_poisoned(other):
+            poison = modifiers.Poison(self, other, 3, self.poison_nova)
+            other.modifier_list.append(poison)
+
+    def check_if_poisoned(self, other):
```

Comment:
check_if_poisoned powinno zwracać True/False - tak wynika z nazwy. A ona robi coś w środku i zwraca False albo None. Wartość logiczna None to też jest False, więc masz metodę, której wartość logiczna zawsze jest False, a czasami jeszcze coś robi... fuj! 
Tak naprawdę możesz to wszystko zrobić w jednej metodzie - poison: przeszukaj listę - jak znajdziesz to zwiększ duration, a jak nie znajdziesz to dodaj nowy modifier.

Reply:
done